Src/FluentAssertions/AssertionExtensions.cs

```diff
@@ -690,7 +690,7 @@ public static FunctionAssertions<T> Should<T>(this Func<T> func)
         }
 
 
-#if NET45 || NET47 || NETCOREAPP2_0
+#if NET45 || NET47 || NETCOREAPP2_0 || NETCOREAPP3_0
```

Comment:
Should we exclude what is not supported instead include?

Reply:
I agree with this.
Blacklisting instead of whitelisting makes adding new target frameworks easier.